### PR TITLE
feature/HTDT-380-add-top-10-feature

### PR DESCRIPTION
**Pull request by evgenybutkevich**

### CODE REVIEW

File: express/services/players.js [hunk 1]
```diff
@@ -82,11 +85,26 @@ function remove(id) {
     });
 }
 
+function getBestPlayers() {
+    const rawQuery = `SELECT DISTINCT ON ("team"."teamName") "teamName",
```
Comment (Egorrr) [inline express/services/players.js]: This query can be moved to the const in the top of the file. 
Please style it properly, indents are confusing. You can install dbeaver app to manage databases and format scripts, or use some online SQL formatting tool

File: sequelize/migrations/20201103113401-create-player.js [hunk 2]
```diff
@@ -15,6 +15,9 @@ module.exports = {
                 allowNull: false,
                 type: DataTypes.STRING
             },
+            avatar: {
+                type: DataTypes.STRING
+            },
```
Comment (Egorrr) [inline sequelize/migrations/20201103113401-create-player.js]: Why have you edited the old migration file, if you want to modify a database schema create a new migration for that

File: sequelize/seeders/04-demo-player.js [hunk 3]
```diff
@@ -1,45 +1,18 @@
...
-        },
-        {
-            firstName: 'Vova',
-            lastName: 'Linnik'
-        }]);
+        const recordsList = [];
+
+        for (let i = 1; i <= 50; i++) {
+            recordsList.push({
+                firstName: faker.name.firstName(0),
```
Comment (Egorrr) [inline sequelize/seeders/04-demo-player.js:12]: Why do you need to call (0)?

File: sequelize/seeders/04-demo-player.js [hunk 4]
```diff
@@ -1,45 +1,18 @@
...
+                lastName: faker.name.lastName(0),
+                avatar: `https://i.pravatar.cc/150?img=${i}`
```
Comment (Egorrr) [inline sequelize/seeders/04-demo-player.js]: Please move this url to const

File: sequelize/seeders/07-demo-result.js [hunk 5]
```diff
@@ -1,35 +1,21 @@
...
-            redCards: 1,
-            gamesPlayed: 9
-        }]);
+        const recordsList = [];
+
+        function getRandomInteger(min, max) {
+            return Math.floor(Math.random() * (max - min + 1)) + min;
+        }
+
+        for (let i = 1; i <= 50; i++) {
```
Comment (Egorrr) [inline sequelize/seeders/07-demo-result.js]: Please move this records count to const

Comment (evgenybutkevich) [reply sequelize/seeders/07-demo-result.js]: Done

File: sequelize/seeders/04-demo-player.js (hunk 4 above)
Comment (evgenybutkevich) [reply sequelize/seeders/04-demo-player.js]: Done

File: sequelize/seeders/04-demo-player.js (hunk 3 above)
Comment (evgenybutkevich) [reply sequelize/seeders/04-demo-player.js:12]: (method) firstName(gender?: number): string
(method) firstName(gender?: number): string
0 - male
1 - female

File: sequelize/migrations/20201103113401-create-player.js (hunk 2 above)
Comment (evgenybutkevich) [reply sequelize/migrations/20201103113401-create-player.js]: Done

File: express/services/players.js (hunk 1 above)
Comment (evgenybutkevich) [reply express/services/players.js]: Done

File: sequelize/seeders/04-demo-player.js [hunk 6]
```diff
@@ -1,45 +1,19 @@
...
-            lastName: 'Gorokhovich'
-        },
-        {
-            firstName: 'Vova',
-            lastName: 'Linnik'
-        }]);
+        const recordsList = [];
+        const imageUrl = 'https://i.pravatar.cc/150?img=';
+
+        for (let i = 1; i <= 50; i++) {
```
Comment (Egorrr) [inline sequelize/seeders/04-demo-player.js]: Please move 50 and imageUrl to the top of the file and make variables capitalized

Comment (evgenybutkevich) [reply sequelize/seeders/04-demo-player.js]: Done

**Review by Egorrr — APPROVED** [review]

**Pull request merged**